### ISSUE TITLE
Large-Scale Object Detection in the Wild from Imbalanced Multi-Labels 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#44]
# TL;DR

OpenImages has a larger label imbalance than with MS COCO dataset and in addition multiple labels on one object, which is different from conventional object detection tasks. Therefore, They proposed soft-balance sampling, which is interpolate between label aware sampling and usual sampling, and concurrent softmax, which includes label co-occurrence probability. 
![image](https://user-images.githubusercontent.com/24524018/90498583-79e15f80-e183-11ea-97a3-c5bb1e5b40f5.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2005.08455

# Submission Dates(yyyy/mm/dd)
2020/05/18

# Authors and institutions

Junran Peng, Xingyuan Bu, Ming Sun, Zhaoxiang Zhang, Tieniu Tan, Junjie Yan
- University of Chinese Academy of Sciences
- SenseTime Group Limited
- Center for Research on Intelligent Perception and Computing, CASIA 

# Methods

# Results

![image](https://user-images.githubusercontent.com/24524018/90498615-81a10400-e183-11ea-8a05-b52dca46dd4b.png)

# Comments
CVPR2020
